src/main/kotlin/wordle/domain/Words.kt

```diff
@@ -0,0 +1,70 @@
+package wordle.domain
+
+import java.time.LocalDate
+import java.time.temporal.ChronoUnit
+
+class Words(private val values: List<Word>) {
+
+    private val answer: Word = findAnswer()
+    private var answerMap: MutableMap<Char, Int> = HashMap()
```

Comment:
`HashMap`을 사용한 이유가 궁금해요!

Reply:
문자를 비교해서 같은 경우 문자를 소진하기 때문에 문자의 개수를 저장할 필요가 있었습니다. 그렇기 때문에 문자 - 문자의 개수를 key, value로 저장하기 위해 HashMap을 사용했습니다! 이 부분이 궁금했던 거 맞나요?

Reply:
이해했습니다!!

Reply:
코틀린은 읽기 전용 컬렉션과 변경 가능한 컬렉션을 구별해 제공합니다.

```suggestion
    private var answerMap: MutableMap<Char, Int> = mutableMapOf()
```